source/_components/sensor.pilight.markdown

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: "Pilight Sensor"
+description: "Instructions how to integrate pilight sensors within Home Assistant."
+date: 2016-10-08 23:21
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: pilight.png
+ha_category: Sensor
+ha_release: 0.31
+ha_iot_class: depends
+---
+
+
+This `pilight` sensor platform for 433 MHz devices uses a value in the message payload as the sensor value. Unique identifiers (e.g. _uuid_) can be set to distinguish between multile pilight devices. The pilight HA hub has to set up.
```

Comment:
Sure I can use ling forms. What I want to say is, that the pilight hub (a different component) has to be setup to make this pilight sensor work.